lib/cn/kvset.h

```diff
@@ -65,14 +67,15 @@ struct kvset_meta {
     struct kvs_block km_hblk;
     struct blk_list km_kblk_list;
     struct blk_list km_vblk_list;
-    uint64_t km_dgen;
-    uint64_t km_vused;
-    uint32_t km_node_offset;
-    uint32_t km_scatter;
-    uint16_t km_node_level;
-    uint16_t km_compc;
-    bool km_capped;
-    bool km_restored;
+    u64             km_dgen;
+    u64             km_vused;
+    u32             km_node_offset;
+    u32             km_scatter;
+    u16             km_node_level;
+    u64             km_nodeid;
+    u16             km_compc;
+    bool            km_capped;
+    bool            km_restored;
```

Comment:
We're using the standard type names these days rather than the Linux kernel type names...

Reply:
Yeah, I've done that in all the new places... I'll fix these too.